add help message

peccu opened the pull request:
add commands for receiving a directory in /help message

peccu committed:
add help message

add commands for receiving a directory in /help message

nwtgck commented:
Thanks. I have not known zipped data is unziped by `tar xf -`.
I'd like to find the `tar` specification.

peccu committed:
Merge branch 'nwtgck:develop' into add-help-for-tar-receiver

peccu committed:
correct commands and options

- add missing option for tar (`z`)
- change unzip command for zip from tar

peccu commented:
Thank you for your review.

- I updated the option for `tar` command. I missed the `x` option for it.
- I changed the command from `tar` to `unzip` for zip version.
  - `unzip` in macOS and ubuntu 22.04 doesn't supports stdin, so I created temporary file.
  - from unzip manpage
  ```
  Archives read from standard input are not yet supported, except with funzip (and then only the first member of the archive can be extracted).
  ```
- I tested both zip and tar version in macOS (Catalina, 10.15.7; sorry for old version) and Ubuntu 22.04 (in docker container with zip/unzip installed)
